site/_docs/reference.md

```diff
@@ -2681,6 +2681,7 @@ BigQuery's type system uses confusingly different names for types and functions:
 | b | CEIL(value)                                    | Similar to standard `CEIL(value)` except if *value* is an integer type, the return type is a double
 | m s | CHAR(integer)                                | Returns the character whose ASCII code is *integer* % 256, or null if *integer* &lt; 0
 | b o p | CHR(integer)                               | Returns the character whose UTF-8 code is *integer*
+| b | CODE_POINTS_TO_BYTES(integers)                 | Converts *integers*, an array of integers between 0 and 255 inclusive, throws error if any element is out of range
```

Comment:
Maybe a small typo, `an array of integers` -> `an array of integer`

Reply:
I think `an array of integers`· is the correct syntax.